src/main/java/com/three_tech_solutions/slot_app/exceptions/GlobalExceptionHandler.java

```diff
@@ -22,8 +25,27 @@ ResponseEntity<ApiError> handleResponseStatusException(ResponseStatusException r
                         .errorMessage(responseStatusException.getReason())
                         .path(request.getRequestURI())
                         .timestamp(LocalDateTime.now())
-                        .trace(Arrays.asList(responseStatusException.getCause().getStackTrace()))
+                        .build()
+                );
+    }
+    @ExceptionHandler(MethodArgumentNotValidException.class)
+    ResponseEntity<ApiError> handleValidationException(MethodArgumentNotValidException ex, HttpServletRequest request) {
+        List<String> errores = ex.getBindingResult()
+                .getFieldErrors()
+                .stream()
+                .map(error -> error.getField() + ": " + error.getDefaultMessage())
```

Comment:
Quitaria el field, simplemente dejaria el mensaje ya que es redundante, por ejemplo:

`dni: El DNI debe contener solo números`

estamos repitiendo dos veces que el campo del error fue DNI
```suggestion
                .map(error -> error.getDefaultMessage())
```